src/flex.story.jsx

```diff
@@ -0,0 +1,215 @@
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 3</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 4</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 5</div></FlexItem>
+		</Flex>
+	))
+	.add('Column', () => (
+		<Flex direction='column' style={flexParentStyles}>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 1</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 2</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 3</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 4</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 5</div></FlexItem>
+		</Flex>
+	))
+	.add('Wrap', () => (
+		<Annotate notes='Wrapping will not occur if no `FlexItem` children have a `shrink` prop to give them a flex shrink factor'>
+			<Flex wrap style={flexParentStyles}>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 1</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 2</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 3</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 4</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 5</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 6</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 7</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 8</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 9</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 10</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 11</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 12</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 13</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 14</div></FlexItem>
+				<FlexItem shrink style={flexItemStyles}><div style={boxStyles}>Item 15</div></FlexItem>
+			</Flex>
+		</Annotate>
+	))
+	.add('No gutters', () => (
+		<Flex noGutters style={flexParentStyles}>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 1</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 2</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 3</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 4</div></FlexItem>
+			<FlexItem style={flexItemStyles}><div style={boxStyles}>Item 5</div></FlexItem>
+		</Flex>
+	))
+	.add('Justify center', () => (
```

Comment:
Not a blocker, but the justification change isn't really visible in storybook right now; We may need to add a parent container around `Flex` here to visually demonstrate `justify-content` variants